substrate/runtime/democracy/src/lib.rs

```diff
@@ -106,8 +108,33 @@ decl_storage! {
 	}
 }
 
+/// An event in this module.
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize, Debug))]
+#[derive(Encode, Decode, PartialEq, Eq, Clone)]
+pub enum RawEvent<Balance, AccountId> {
+	Tabled(PropIndex, Balance, Vec<AccountId>),
+	Started(ReferendumIndex, VoteThreshold),
+	Passed(ReferendumIndex),
+	Failed(ReferendumIndex),
```

Comment:
Can we come up with a different name for this case? This introduces some ambiguity with the result of dispatching  the proposal. Is there a special word in English for an unsuccessful vote?

Reply:
1. `NotPassed`
2. `KeepStatusQuo`